.chloggen/codeboten_rm-dupe-dockerstats.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Is this really a breaking change if there is no impact to users? Seems to be an enhancement to me.

Reply:
It's a breaking change for the API if anyone was instantiating the configuration for the component programmatically